### PR TITLE
Final fixes

### PR DESCRIPTION
**Pull request by marikaris**
- Add error handling for mismatching def/data file
- Add unittests
- Increase max lines to make sure every point will be plotted
- Realign alert progress box

### CODE REVIEW

File: test/unit/specs/components/SpnDescentPlots.spec.js [hunk 1]
```diff
@@ -6,4 +6,9 @@ describe('SnpDescentPlots', () => {
       expect(SnpDescentPlots.name).to.equal('snp-descent-plot')
     })
   })
+  describe('forEachLine', () => {
+    it('should', () => {
```
Comment (ConnorStroomberg) [inline test/unit/specs/components/SpnDescentPlots.spec.js]: test is missing description

File: test/unit/specs/service/lineReader.spec.js [hunk 2]
```diff
@@ -18,8 +18,9 @@ cleanse kinfolk crucifix marfa gochujang elit chicharrones food truck.`
       const maxLines = 3
       let linesRead = []
       let onCompleteCalled = false
-      const forEachLine = function (line) {
+      const forEachLine = function (line, fileOk) {
```
Comment (ConnorStroomberg) [inline test/unit/specs/service/lineReader.spec.js]: i would suggest using 'isFileOk' in general is suggest using the template is{variableName} for booleans

File: test/unit/specs/components/SpnDescentPlots.spec.js (hunk 1 above)
Comment (marikaris) [reply test/unit/specs/components/SpnDescentPlots.spec.js]: Last commit was not pushed yet, this test is gone now

File: test/unit/specs/service/lineReader.spec.js (hunk 2 above)
Comment (marikaris) [reply test/unit/specs/service/lineReader.spec.js]: i will change this

**Pull request merged**